src/styles/stopwatch.scss

```diff
@@ -0,0 +1,73 @@
+.stopwatch {
+  position: absolute;
+  top: 0;
+  right: 0;
+  left: 0;
+  bottom: 0;
+  margin: auto;
+  background-color: #ffd700;
+  border: 4px #daa520;
+  border-color: #daa520;
+  box-shadow: 10px;
+  height: 500px;
+  width: 500px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 100%;
+
+  &::before {
+    content: "";
+    position: absolute;
```

Comment:
it is centered because of the `flexbox`
So no need to add  `position: absolute;` here. Or vice versa - you can remove `display: flex` from `.stopwatch`